### PR TITLE
Logout bug

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
- https://www.pivotaltracker.com/n/projects/1662141

#### Brief feature description
- user logout bug was not passing props

#### Does this PR block/effect other stories?
- no 

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

File: app/javascript/header/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default class HeaderComponent extends React.Component {
   render () {
     return (
       <div>
-        <GlobalHeader />
+        <GlobalHeader {...this.props}/>
```
Comment (ratneshraval) [inline app/javascript/header/index.js]: Can you explicitly specify each prop instead of ...props

**Comment by ratneshraval** [comment]
@Nasir2712 can you add a test for headerComponent too?

File: app/javascript/header/index.js (hunk 1 above)
Comment (Nasir2712) [reply app/javascript/header/index.js]: yup

**Comment by Nasir2712** [comment]
@ratneshraval  added a test case and addressed your comments.

**Pull request merged**